MidCamp/midcamp
# setup moderation and roles

Turn on content moderation
Add roles
Setup workflow for topics Draft > Submitted > Accepted

## Testing instructions
* `drush cim -y`
* Masquerade as an authenticated user
* Navigate to /node/add/topic
* Observe you can create a topic
* Observe you can Save as "Draft"
* Observe you can Save as "Submitted"
* Masquerade as a content editor
* Navigate to a Topic
* Observe you can Save as "Accepted"
* Observe you can use the administration theme
* Observe you can create and edit any content
* Observe you can create and edit any terms
